AmplifyPlugins/API/Sources/AWSAPIPlugin/Core/AppSyncListPayload.swift

```diff
@@ -43,7 +46,7 @@ public struct AppSyncListPayload: Codable {
         // `GraphQLFilter`.
         guard let filterVariablesData = try? encoder.encode(filterValue),
               let filterVariablesJSON = try? JSONSerialization.jsonObject(with: filterVariablesData)
-                as? GraphQLFilter else {
+                as? [String: Any] else {
```

Comment:
why this change?

Reply:
i've reverted it, when i was compiling earlier Xcode wasn't picking up GraphQLFilter properly from AWSPluginsCore